src/disco/reducers/installations.js

```diff
@@ -0,0 +1,51 @@
+import {
+  DOWNLOADING,
+  ERROR,
+  INSTALLED,
+  INSTALLING,
+  UNINSTALLED,
+  UNINSTALLING,
+} from 'disco/constants';
+
+export default function installations(state = {}, { type, payload }) {
+  const newState = {...state};
+  if (type === 'INSTALL_STATE') {
```

Comment:
I originally used a switch statement but you can't define a `const` in a `case`. If I move `addon` to the top that would work.